gallery/visualtests/nxCheckbox.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { clickTest, focusTest, focusAndHoverTest, hoverTest, simpleTest } = require('./testUtils');
+
+describe('NxCheckbox', function() {
+  beforeEach(async function() {
+    await browser.url('#/pages/NxCheckbox');
+  });
+
+  describe('Default NxCheckbox', function() {
+    const selector = '#nx-checkbox-default-example';
+
+    it('has a light grey border and white background by default', simpleTest(selector));
+    it('has a black border when hovered', hoverTest(selector));
+    it('has a thick blue border and white background when clicked', clickTest(selector));
+    it('has a light blue border and glow when focused', focusTest(selector));
+    it('has a light blue border and glow when focused and hovered', focusAndHoverTest(selector));
```

Comment:
This test name isn't currently correct to what it should do

Reply:
> This test name isn't currently correct to what it should do

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/119/commits/3c9987412a547f01f97a4d8230adb0a7b138f474